src/components/MessageBar/MessageBar.scss

```diff
@@ -13,11 +13,16 @@ $MessageBar-padding: 8px;
   @include ms-baseFont;
   padding: $MessageBar-padding;
   @include ms-bgColor-info;
+  color: $ms-color-infoText;
   width: 100%;
   box-sizing: border-box;
   display: flex;
   position: relative;
 
+  .ms-MessageBar-icon {
+    color: $ms-color-info;
```

Comment:
This way it's closer to where we set the default text color and background color. I wanted all the default colors to be set here.